Makefile

```diff
@@ -50,50 +43,44 @@ clean: ## Make a clean source tree
 	-find website -name '.sass-cache' -type d -exec rm -fr {} +
 	-find website -name '_site' -type d -exec rm -fr {} +
 	-find website -name 'build' -type d -exec rm -fr {} +
-	-$(SA) $(ENV) && make -C docs clean
-	-$(SA) $(ENV) && make -C etc clean
+	-make -C docs clean
+	-make -C etc clean
 
 lint: ## Check code style
-	$(SA) $(ENV) && pre-commit run --all-files
+	pre-commit run --all-files
 
-nuke: ## Make clean + remove conda env
-	-conda env remove -n $(ENV) -y
-
-dev: ## Make a server in jupyter_websocket mode
-	$(SA) $(ENV) && python enterprise_gateway
+run-dev: test-install-wheel ## Make a server in jupyter_websocket mode
+	python enterprise_gateway
 
 docs: ## Make HTML documentation
-	$(SA) $(ENV) && make -C docs requirements html
+	make -C docs requirements html
 
 kernelspecs:  kernelspecs_all kernelspecs_yarn kernelspecs_conductor kernelspecs_kubernetes kernelspecs_docker kernel_image_files ## Create archives with sample kernelspecs
 kernelspecs_all kernelspecs_yarn kernelspecs_conductor kernelspecs_kubernetes kernelspecs_docker kernel_image_files:
 	make VERSION=$(VERSION) TAG=$(TAG) SPARK_VERSION=$(SPARK_VERSION) -C  etc $@
 
-install: ## Make a conda env with dist/jupyter_enterprise_gateway-*.whl and dist/jupyter_enterprise_gateway-*.tar.gz installed
```

Comment:
Could we still leave the `install` at least as an undocumented task?

Reply:
Not sure why - it has no value other than to test installs.   From the PR description:
> While doing this, I found that the former make install target was more like a test-install target, so I named it as such and split the two forms of installation it was performing into separate targets (so we can make use of one from our CI actions).

I didn't realize this, but it was doing two installations, one via the wheel, the other via the tar file, and **uninstalling** both after each step.  Nothing more than a test target.

Reply:
I was under the impression that, once we run a  `make install` we build and install the package on the current env, no?

Reply:
> I was under the impression that, once we run a make install we build and install the package on the current env, no?

See the deleted code immediately below. It...

0. deletes the "install" conda env
1. creates the "install" conda env
2. activates the env
3. installs the wheel file
4. runs EG --help
5. uninstalls EG
6. removes the "install" conda env
then repeats 1-6 using the tar file.

That feels like an "install test" to me, so I renamed the target and broke each loop into separate targets, and eliminated the creation/removal of the "install" conda env.

I suppose we can add a `make install` target to invoke `pip install`, but the `make install` target you've been using doesn't wind up installing anything permanent.  I tend to copy the `whl` file to a cluster or build it into an image, so I rarely install EG where I build it.  But when I need to, I use use `pip install`.

> And thus, without that, I will need to run make build, and look for the wheel to install it?

The distributions are always in `dist` and the wheel is the only `whl` extension. You'd also need to deal with the kernelspec tars in that same directory.